Example/Okta_UITests/UITestUtils.swift

```diff
@@ -19,11 +19,22 @@ public struct UITestUtils {
     }
 
     func login(username: String, password: String) {
+        // Wait for browser to load
+        // This sleep bypasses the need to "click" the consent for Safari
+        sleep(5)
+
+        // Known bug with iOS 11 and system alerts
+        testApp.tap()
+
         // Login via username and password inside of the Safari WebView
         let webViewsQuery = testApp.webViews
         let usernameField = webViewsQuery.textFields["Username"]
         let passwordField = webViewsQuery.secureTextFields["Password"]
 
+        if !waitForElement(usernameField, timeout: 10) && waitForElement(passwordField, timeout: 10) {
```

Comment:
Wondering if you want exclamation point negation in front of both waitForElement()s here?